checkov/main.py

```diff
@@ -112,7 +113,7 @@ def exit_run(no_fail_on_crash: bool) -> None:
     exit(0) if no_fail_on_crash else exit(2)
 
 
-def commit_repository(config: Namespace) -> Optional[str]:
+def commit_repository(config: Namespace) -> str | None:
```

Comment:
What's the benefits of using `| None` instead of `Optional`?

Reply:
at the end it is the same, you just don't need to import `Optional`. This is current preferred way of defining type hint for a possible `None` value.